app/views/podcasts/_form_main.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="col-12 mb-4">
   <div class="form-floating input-group">
     <%= form.text_field :title %>
-    <%= form.label :title %>
+    <%= form.label :title, required: "true" %>
```

Comment:
This is likely temporary until. I'm assuming theres a way to get this into the model.

Reply:
I'm probably okay with them being in the views.  But we'll see.